client/src/client.rs

```diff
@@ -1109,6 +1109,28 @@ pub mod asset {
     }
 }
 
+pub mod block {
```

Comment:
@appetrosyan I'd need to add these queries to the tutorial, right?

Reply:
One subtlety is that the tutorial now needs to serve either the up to date guide, or the guide that is appropriate for LTS. Right now we have no process for separating one from the other.